packages/common/src/components/AutosizeInput/index.test.tsx

```diff
@@ -0,0 +1,16 @@
+import 'jest-styled-components';
+import React from 'react';
+import mountWithTheme from '../../test/themeMount';
+import AutosizeInput from '.';
+
+let wrapper;
+
+beforeEach(() => {
+  wrapper = mountWithTheme(<AutosizeInput />);
```

Comment:
I think it would be more consistent with the other tests if you do this inside the `it` callback.